src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -91,16 +93,19 @@ public CommandResult execute(Model model) throws CommandException {
      * Creates and returns a {@code Person} with the details of {@code personToEdit}
      * edited with {@code editPersonDescriptor}.
      */
-    private static Person createEditedPerson(Person personToEdit, EditPersonDescriptor editPersonDescriptor) {
+    private static Person createEditedPerson(Student personToEdit, EditPersonDescriptor editPersonDescriptor) {
```

Comment:
I think technically the cleaner way to do this is to extend something from the edit command class called EditStudentCommand or something, because when I edited this code, all commands were poised to take in Person as a parameter, not Student. I lazily changed the command to accept Student first, but at some point i guess we would have to extend the command class.